src/github-repo-event-notifier.coffee

```diff
@@ -32,12 +32,16 @@
 #   patcon
 #   parkr
 
+_ = require('lodash')
 inspect = (require('util')).inspect
 url = require('url')
 querystring = require('querystring')
-eventActions = require('./event-actions/all')
 eventTypesRaw = process.env['HUBOT_GITHUB_EVENT_NOTIFIER_TYPES']
 eventTypes = []
+customActions = {}
+try
+  customActions = require.main.require('../../../scripts/github/custom-actions')
```

Comment:
@parkr would you be open to use something like [**findup-sync**](https://www.npmjs.com/package/findup-sync)?

That's the closest thing I found regarding accessing `node_module`'s parent directories.

Let me know if you are aware of another method/lib, otherwise I can update the PR. 😄